.github/workflows/test-gha-eks.yml

```diff
@@ -92,6 +104,20 @@ jobs:
                   fi
                   echo "db_password=$db_password" | tee -a "$GITHUB_OUTPUT"
 
+                  if [[ -n "${{ inputs.opensearch_username }}" ]]; then
+                    opensearch_username="${{ inputs.opensearch_username }}"
+                  else
+                    db_username="user$(openssl rand -hex 4 | tr -d '/@" ')"
```

Comment:
the variable name looks wrong if I look at the surrounding code and don't see it used anymore.

`db_username` vs `opensearch_username`.